gig-marketplace.md

```diff
@@ -0,0 +1,57 @@
+## Title
+
+* Gig Marketplace
+
+## Patlet
+
+InnerSource projects at your company are finding it difficult to attract contributors.  
+
+Establish a marketplace by creating an intranet website that list specific InnerSource project needs as "gigs" with explicit time and skill requirements.
+This will enable managers to better understand their employee’s time commitment and professional benefits thereby increasing the likelihood of garnering approval to make InnerSource contributions.
+
+## Problem
+
+There is no single, uniform way for InnerSource project owners and potential contributors to discover each other. 
```

Comment:
Does the marketplace help you to discover specific projects, or is it more like discovering _tasks of a certain type_ (no matter to what project they belong)?

Reply:
That's correct.  The InnerSource portal pattern is meant to address the project discovery aspect.  The Gig Marketplace pattern is centered around organizing potential contribution opportunities into categories mapping back to individual developer's interests.